packages/common/src/testUtil/testSubsetGrep.txt

```diff
@@ -0,0 +1,12 @@
+# This file contains the grep strings to pass to Mocha when running a subset of tests.
+# These grep strings will be used with the "Run Single Extension Test" launch configuration.
+# See https://mochajs.org/#-grep-regexp-g-regexp for supported syntax.
+#
+# One regular expression per line.
+# Tests matching any of the regular expressions will be run.
+#
+# To ask git not to track changes to this file, run:
+#   git update-index --skip-worktree packages/common/src/testUtil/testSubsetGrep.txt
```

Comment:
Isn't there a way we can automate this so that each developer doesn't have to run this command?
Otherwise we can just use that command on the current `runTestSubset.ts` file?

Reply:
Unfortunately, you can't git ignore an existing file. I thought you could ( Maybe in the past you could?)  But it doesn't work. in classic git style, this mouthful is the recommendedapproach.

 an alternative is to git ignore the path and then create the file the first time any tests get run.   Assuming that the user runs the tests before they ever try to read the documentation and change this file, it will work like magic.

yet another alternative is to just document this git command. Although in that case it is still worth splitting out the easily editable parts into a separate file so you can just ignore that part.

I'm open to any approach as long as it leaves me with a clean tree. :)

Reply:
I'd be inclined to not have the file in source control at all.  I'd argue to either

- generate the file, or
- just show them a command in the docs that they can use to generate it, eg `echo "fooBar" > the/path/to/the/file.txt`, or
- we could map a voice command in our dev talon files

Reply:
I would prefer for the file to be generated. Adding additional voice commands for something that's gonna be done once feels unnecessary.

Reply:
OK, unless I hear otherwise in the next day or two, I will generate the file whenever a test command is run.

Reply:
Sure I'm fine with generating it, as long as that workflow is documented